cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -249,6 +281,20 @@ func (i *deployCommand) Exec(cmd *cobra.Command, args []string) (err error) {
 	if hasSource {
```

Comment:
`fromFile` and `hasSource` it's almost the same thing semantically. Can we think on another naming/approach?